packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -325,6 +325,24 @@ class UiKitView extends _DarwinView {
   State<UiKitView> createState() => _UiKitViewState();
 }
 
+/// Embeds a MacOS view in the Widget hierarchy.
+class AppKitView extends _DarwinView {
+  /// Creates a widget that embeds a macOS view.
```

Comment:
Maybe to be slightly more specific:
```suggestion
  /// Creates a widget that embeds a macOS AppKit NSView.
```